libvcx/src/aries/handlers/proof_presentation/verifier/state_machine.rs

```diff
@@ -44,10 +43,10 @@ impl VerifierSM {
 
         for (uid, message) in messages {
             match self.state {
-                VerifierState::Initiated(_) => {
+                VerifierFullState::Initiated(_) => {
```

Comment:
We could use `VerifierState` to make this code cleaner

Reply:
As I said above, I would prefer consistency / symmetry over such a minor improvement in readability.